.ipynb_checkpoints/SimpleQueue.py

```diff
@@ -26,7 +26,6 @@ def size(self):
     def clear(self):
         self.end = 0
         self.start = 0
-        self.array = self.Size*[0]
```

Comment:
Global comment:

This code will not work if I add 100 elements, then pop 100 elements and add 100 elements again